test/acceptance_with_go_client/search_test.go

```diff
@@ -195,9 +195,12 @@ func TestSearchOnSomeProperties(t *testing.T) {
 				alpha = "alpha:0" // exclude vector search, it doesn't matter for this testcase
 			}
 
-			results, err := c.GraphQL().Raw().WithQuery(fmt.Sprintf("{Get{%s(%s:{query:\"hello\", properties: [\"%s\"] %s} ){_additional{id}}}}", className, tt.queryType, tt.property, alpha)).Do(ctx)
+			results, err := c.GraphQL().Raw().WithQuery(fmt.Sprintf("{Get{%s(%s:{query:\"hello\", properties: [\"%s\"] %s} ){_additional{id score}}}}", className, tt.queryType, tt.property, alpha)).Do(ctx)
 			result := results.Data["Get"].(map[string]interface{})[className].([]interface{})
 			require.Len(t, result, tt.results)
+			if len(result) > 0 && fmt.Sprintf("%v", result[0].(map[string]interface{})["score"]) == "0" {
```

Comment:
can you convert that to a number and then compare? It currently would suceed in case 0.0 is returned which is what the bugreport is claiming